pkg/provider/azure_loadbalancer.go

```diff
@@ -2919,17 +2814,12 @@ func (az *Cloud) reconcileSecurityGroup(
 		WithValues("delete-lb", !wantLb)
 	logger.V(2).Info("Starting")
 
+	ctx := klog.NewContext(context.Background(), logger)
```

Comment:
How about using context from outside caller?

Reply:
I am thinking so, will do it in another logging refactor PR.